Fix in ai-battlefield.md: fix list format in FLOPS section and change Flops to FLOPS in MFU section (#50)
stas00/ml-engineering: insights/ai-battlefield.md

Emerging to general availability:

>   * NVIDIA H100 - 2-3x faster than A100 (half precision), 6x faster for fp8 - H200 has already been released (more faster HBM memory)

Reviewer: ```suggestion
  * NVIDIA H100 - 2-3x faster than A100 (half precision), 6x faster for fp8. H200 with more and faster HBM memory is starting to become available as well.
```

   When was the last time you went to the bread section of the supermarket and found a lump of half-baked dough with a note "we made a reasonable effort to bake this bread, but alas, what you see is what you get"?

>    But for whatever reason it's unacceptable to create a legal contract where the provider provides neither delivery dates nor performance metrics and doesn't provide stipulations for what will they do in recompense when those promises aren't fulfilled.

Reviewer: ```suggestion
   But for whatever reason it's acceptable to create a legal contract where the provider provides neither delivery dates nor performance metrics and doesn't provide stipulations for what will they do in recompense when those promises aren't fulfilled.
```

reverting this one since it was intentional.

- Obviously if you choose compute that requires custom software that works for that hardware only and you can't rent this hardware anywhere else you're setting yourself up for a lock-in

### Don't buy what you don't really need

- The cloud providers have mostly the same generic hardware, which leads to a very slim $$ margin and so in order to make big $$ they invent products and then try to convince you that you need to buy them. Sometimes you actually need those products, but very often not. See also the previous section on lock-in, since proprietary products usually mean a partial lock-in.

- Often it's easy to observe the 3 step marketing technique for solutions that seek a problem to solve:

>   1. Convince a couple of well respected customers to use the provider's proprietary products by giving them huge discounts or even pay them to use them
>   2. Use those in step 1 as the social approval lever to reel in more converts
>   3. Then scoop the rest of the strugglers by telling them that 80% of your customers (1+2) use these amazing products

Reviewer: ```suggestion

  1. Convince a couple of well respected customers to use the provider's proprietary products by giving them huge discounts or even pay them to use them
  2. Use those in step 1 as the social approval lever to reel in more converts
  3. Then scoop the rest of the strugglers by telling them that 80% of your customers (1+2) use these amazing products
```
